CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java

```diff
@@ -282,22 +280,22 @@ public boolean needCleanTestCase()
 	
 	public String getTestCaseTimeout()
 	{
-		return getProperty("main.testcase.timeout", "-1");
+		return getProperty(ConfigParameterConstants.TESTCASE_TIMEOUT_IN_SECS, "-1");
 	}
 	
 	public boolean needEnableMonitorTrace()
 	{
-		return com.navercorp.cubridqa.common.CommonUtils.convertBoolean(getProperty("main.monitor.enable_tracing", "false"));
+		return com.navercorp.cubridqa.common.CommonUtils.convertBoolean(getProperty(ConfigParameterConstants.ENABLE_STATUS_TRACE_YES_OR_NO, "false"));
 	}
 	
 	public String getExcludedCoresByAssertLine()
 	{
-		return getProperty("main.testing.excluded_cores_by_assert_line");
+		return getProperty(ConfigParameterConstants.IGNORE_CORE_BY_KEYWORDS);
 	}
 	
 	public boolean needDeleteTestCaseAfterTest()
 	{
-		return getProperty("main.testing.delete_test_case_after_execution", "false").trim().toLowerCase().equals("true");
+		return getProperty(ConfigParameterConstants.DELETE_TESTCASE_AFTER_EACH_EXECUTION_YES_OR_NO, "false").trim().toLowerCase().equals("true");
```

Comment:
Please use convertBoolean to convert ConfigParameterConstants.DELETE_TESTCASE_AFTER_EACH_EXECUTION_YES_OR_NO